src/web/app/src/main/resources/geostore-ovr.properties

```diff
@@ -23,4 +23,31 @@
 #geostoreInitializer.allowPasswordRecoding=true
 
 # Configure session duration.
-#restSessionService.sessionTimeout=3600
\ No newline at end of file
+#restSessionService.sessionTimeout=3600
+
+#OpenId PropertiesO
+
+## mandatory props
+googleOAuth2Config.clientId=
+googleOAuth2Config.clientSecret=
+googleOAuth2Config.autoCreateUser=true
+googleOAuth2Config.redirectUri=
+googleOAuth2Config.internalRedirectUri=
+googleOAuth2Config.enabled=false
+
+ ## needed if groups and roles should be extracted from the claims of the id_token.
+googleOAuth2Config.groupsClaim=
+googleOAuth2Config.rolesClaim=
+
+
+## if not specified the optional properties  needs to be populated
+googleOAuth2Config.discoveryUrl=https://accounts.google.com/.well-known/openid-configuration
+
+## the following are optional and auto-populated if the discoveryUrl is provided
+googleOAuth2Config.accessTokenUri=
+googleOAuth2Config.authorizationUri=
+googleOAuth2Config.checkTokenEndpointUrl=
+googleOAuth2Config.logoutUri=
+googleOAuth2Config.scopes=
+googleOAuth2Config.idTokenUri=
```

Comment:
do not add anything to `geostore-ovr.properties` It may be confusing. We should instead document the tecnique.